modules/extensions/src/main/java/org/apache/synapse/mediators/bsf/ScriptMediator.java

```diff
@@ -71,6 +77,7 @@
  * does not then true is assumed.
  */
 public class ScriptMediator extends AbstractMediator {
```

Comment:
Document option to use "nashhornJs" with this change.

Reply:
Hi,
Fixed with https://github.com/wso2/wso2-synapse/pull/971